tools/ci/choose_workflows.py

```diff
@@ -41,6 +41,7 @@ def should_run_sr_unit_tests(ctx: CIContext) -> bool:
         pr_keyword='[x] Run unit tests for Session Replay',
         pr_path_prefixes=[
             'DatadogSessionReplay/',
+            'TestUtilities/',
```

Comment:
If also adding this to `should_run_unit_tests` definition, we will be fully set 😉:
* if a PR makes change to SR and TU → run SDK unit tests for sanity check
* if a PR makes change to SDK and TU → run SR unit tests for sanity check

Otherwise, changing SR and TU might break SDK units and we won't notice this on PR branch.